src/bundle/Resources/public/scss/_main-row.scss

```diff
@@ -0,0 +1,36 @@
+.ez-main-row {
+    display: grid;
+    grid-template-columns: 1fr calculateRem(80px);
+    grid-template-rows: auto;
+    grid-template-areas: "ez-content-container ez-context-menu";
+
+    .ez-content-container {
+        grid-area: ez-content-container;
+    }
+
+    .ez-context-menu {
+        grid-area: ez-context-menu;
+    }
+}
+
+.ez-content-view,
+.ez-bookmark-list-view,
+.ez-trash-list-view,
+.ez-drafts-list-view,
+.ez-search-view {
```

Comment:
Maybe you should introduce a new CSS class instead of styling by many parent selectors? Something like `ez-has-two-sidebars` or something similar? This class could be added to the `body` tag.